homeassistant/components/tahoma/cover.py

```diff
@@ -228,22 +229,22 @@ def stop_cover(self, **kwargs):
             == "io:RollerShutterWithLowSpeedManagementIOComponent"
         ):
             self.apply_action("setPosition", "secured")
-        elif self.tahoma_device.type in (
-            "rts:BlindRTSComponent",
+        elif self.tahoma_device.type in {
```

Comment:
I am curious why do you use sets instead of tuples?

Reply:
Because of this feedback: https://github.com/home-assistant/home-assistant/pull/28650#discussion_r344432103

Reply:
Good to know, thanks for the explanation.